src/ansys/additive/core/parametric_study/parametric_utils.py

```diff
@@ -38,7 +38,7 @@ def build_rate(
         Laser scan speed.
     layer_thickness : float
         Powder deposit layer thickness.
-    hatch_spacing : float, None
+    hatch_spacing : float, default: None
```

Comment:
Maybe change these to "default None" if you want it to be the same across files (referring to one of the comments above)

Reply:
adopted `default: None` across all files.